.travis.yml

```diff
@@ -5,6 +5,6 @@ env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
 rvm:
-  - 2.1
```

Comment:
Are we dropping support for 2.1? I seem to remember that 2.1 may be the latest ruby available through some package managers?